rdt-open-reader/src/covidIndonesia/assets/json.form/patient-diagnostics-form.json

```diff
@@ -814,6 +877,28 @@
           "value": "true",
           "err": "{{patient-diagnostics-form-original.step2.has_fatigue.v_required.err}}"
         }
+      },
+      {
+        "key": "has_feel_malaise",
```

Comment:
Could we rename the key to `feels_malaised` instead?

Reply:
fixed.